src/pages/ChessPage/ChessPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import axios from "axios";
+import { extractLichessGameIds } from "../../utils/lichessUtils";
+import RecentGamesContent from "../../components/Tab/TabContent/RecentGamesContent";
+import { numberOfRecentGamesShown } from "../../constants/lichessConstants";
+import { ChessPageTabs } from "../../components/Tab/tabConstants";
+import TabButton from "../../components/Tab/TabButton";
+
+import './ChessPage.css';
+import LinksContent from "../../components/Tab/TabContent/LinksContent";
+
+const ChessPage = () => {
+    const [games, setGames] = React.useState<String>("");
+    const [gamesIds, setGamesIds] = React.useState<String []>([]);
+    const [loaded, setLoaded] = React.useState<Boolean>(false);
+
+    const [displayTab, setDisplayTab] = React.useState<number>(ChessPageTabs.RecentGames);
+
+    React.useEffect(() => {
+        const fetchGames = async () => {
+          try {
+            const response = await axios.get(
+                `https://lichess.org/api/games/user/guygreenInClassAtUCT?max=${numberOfRecentGamesShown}`
+            );      
+            setGames(response.data);
+          } catch (error) {
+                console.error("Error fetching games:", error);
+          }
+        };
+    
+        fetchGames();
+      }, []);
+   
+    React.useEffect(() => {
+        if (games?.length > 0) {
+            setGamesIds(extractLichessGameIds(games));
+        }
+    }, [games]);
+
+
+    if (gamesIds.length === numberOfRecentGamesShown && !loaded) {
```

Comment:
What if someone copies this code for their own account, but they have less games than numberOfRecentGames shown